samples/scala-valueentity-counter/src/main/scala/com/example/actions/DoubleCounterAction.scala

```diff
@@ -0,0 +1,47 @@
+/* This code was generated by Akka Serverless tooling.
+ * As long as this file exists it will not be re-generated.
+ * You are free to make changes to this file.
+ */
+package com.example.actions
+
+import com.akkaserverless.scalasdk.SideEffect
+import com.akkaserverless.scalasdk.action.Action
+import com.akkaserverless.scalasdk.action.ActionCreationContext
+import com.example.IncreaseValue
+import com.google.protobuf.empty.Empty
+
+// tag::controller-forward[]
+// tag::controller-side-effect[]
+/** An action. */
+class DoubleCounterAction(creationContext: ActionCreationContext) extends AbstractDoubleCounterAction {
```

Comment:
@RayRoestenburg note that some of the samples for actions docs is already included here